### PR TITLE
some fixes building without X stuff.

### PR DESCRIPTION
**Pull request by hiro2233**
Tested and working wxWidgets natively on wayland with drm backend and Opengl ES 2.0 with a Raspberry Pi2.

Examples working:

- isosurf    OK
- penguin  OK
- cube       OK

pyramid  example throw an error beacause Opengl version mismatch.

### CODE REVIEW

**Comment by vadz** [comment]
Looks good, thanks! Will merge once the CI builds finish.

File: src/gtk/window.cpp [hunk 1]
```diff
@@ -2460,7 +2460,7 @@ wxWindow *wxGetActiveWindow()
 
 // Under Unix this is implemented using X11 functions in utilsx11.cpp but we
 // need to have this function under Windows too, so provide at least a stub.
-#ifndef GDK_WINDOWING_X11
+#if !defined(GDK_WINDOWING_X11) && defined(__WXMSW__)
```
Comment (paulcor) [inline src/gtk/window.cpp:2463]: I don't think `__WXMSW__` is ever defined when building wxGTK.
```suggestion
#ifdef GDK_WINDOWING_WIN32
```

**Comment by vadz** [comment]
There are some real problems in the Travis CI builds, could you please fix them? TIA!

**Comment by hiro2233** [comment]
@vadz Ok, no problem, today i will fix them.

**Comment by vadz** [comment]
@hiro2233 is it ok to merge it now (note that I've cancelled the AppVeyor build as wxMSW shouldn't be affected by these changes) or do you have any other planned changes?

**Comment by hiro2233** [comment]
@vadz it's ok, go ahead. I'm focused on wxGTK and OpenGL features on unix like environments, but i will send another pr with some wxGTK 3 build fixes on msys2, maybe someone wants to run on broadway on windows.

**Comment by vadz** [comment]
Merged now, thanks again!

**Pull request closed without merging**